int32 obj_num
obj_tracking[] data
float32 car_x
float32 car_y